vagrant-kubernetes-istio/Vagrantfile

```diff
@@ -0,0 +1,89 @@
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  # config.vm.box_check_update = false
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine. In the example below,
+  # accessing "localhost:8080" will access port 80 on the guest machine.
+  # NOTE: This will enable public access to the opened port
+  config.vm.network "forwarded_port", guest: 5000, host: 5000, host_ip: "10.10.0.2"
+  config.vm.network "forwarded_port", guest: 8080, host: 8080
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine and only allow access
+  # via 127.0.0.1 to disable public access
+  # config.vm.network "forwarded_port", guest: 80, host: 8080, host_ip: "127.0.0.1"
+
+  # Create a private network, which allows host-only access to the machine
+  # using a specific IP.
+  config.vm.network "private_network", ip: "10.10.0.2"
+
+  # Create a public network, which generally matched to bridged network.
+  # Bridged networks make the machine appear as another physical device on
+  # your network.
+  # config.vm.network "public_network"
+
+  # Share an additional folder to the guest VM. The first argument is
+  # the path on the host to the actual folder. The second argument is
+  # the path on the guest to mount the folder. And the optional third
+  # argument is a set of non-required options.
+  config.vm.synced_folder "~/go/src/istio.io", "/istio.io"
+
+  # Provider-specific configuration so you can fine-tune various
+  # backing providers for Vagrant. These expose provider-specific options.
+  # Example for VirtualBox:
+  #
+  config.vm.provider "virtualbox" do |vb|
+     # Display the VirtualBox GUI when booting the machine
+  #   vb.gui = true
+  #
+     # Customize the amount of memory on the VM:
+     vb.memory = "8192"
+     vb.cpus = "4"
+     # Set the vboxnet interface to promiscous mode so that the docker veth
+     # interfaces are reachable
+     vb.customize ["modifyvm", :id, "--nicpromisc2", "allow-all"]
+     # Otherwise we get really slow DNS lookup on OSX (Changed DNS inside the machine)
+     vb.customize ["modifyvm", :id, "--natdnsproxy1", "on"]
+  end
+  #
+  # View the documentation for the provider you are using for more
+  # information on available options.
+
+  # Enable provisioning with a shell script. Additional provisioners such as
+  # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
+  # documentation for more information about their specific syntax and use.
+  config.vm.provision "shell", inline: <<-SHELL
+     echo "provisioning vagrant vm..."
+     apt-get update
+     echo "install git"
+     sudo apt-get install --quiet -y git
+     echo "install go"
+     sudo curl -O https://dl.google.com/go/go1.10.1.linux-amd64.tar.gz
```

Comment:
Is it possible to get the link of the latest release version like kubectl?

Reply:
Doesn't seem like there is a way here: https://golang.org/dl/

Reply:
I find a way to download it. Thanks for the advice.